source/docs/advanced/production-ready/logging.md

```diff
@@ -144,3 +144,21 @@ module.exports = {
   ],
 };
 ```
+
+## Automatic logging configuration
+
+You can also automate the logging configuration based on wether the application is deployed on our cloud offer (V2) or not using the following `my-module/config/logging.js`
+
+```js
+const makeLogAccordingToEnv = (filename) =>
+  process.env.FRONT_COMMERCE_CLOUD_VERSION === "2"
+    ? { type: "console" }
```

Comment:
a little explanation of why they should use console in the cloud V2 will be helpful

Reply:
on the cloud V2 we stream the logs to the standard output to then gather and exploit them in another platform